bin/main.rb

```diff
@@ -1,64 +1,44 @@
 #!/usr/bin/env ruby
 
-# Array of valid positions that a player can take
-valid_options = %w[1 2 3 4 5 6 7 8 9]
+require './lib/board'
+require './lib/validation'
```

Comment:
Change this to `require_relative` and folder location is wrong, fix this before merge the project